packages/js/src/ui/components/Notification/DefaultNotification.tsx

```diff
@@ -214,9 +215,15 @@ export const DefaultNotification = (props: DefaultNotificationProps) => {
           </div>
         </div>
         <Show when={props.notification.subject}>
-          <p class={style('notificationSubject', 'nt-text-start nt-font-semibold')}>{props.notification.subject}</p>
+          {(subject) => (
```

Comment:
Use the function here since as that's the correct way to handle this.
https://docs.solidjs.com/reference/components/show